handler/handler.go

```diff
@@ -375,7 +375,19 @@ func (h *Handler) Run() error {
 				ctx, stop := signal.NotifyContext(context.Background(), os.Interrupt)
 				if err = h.Execute(ctx, out, opt, h.lastPrefix, h.last, forceBatch); err != nil {
 					lastErr = WrapErr(h.last, err)
-					fmt.Fprintln(stderr, "error:", err)
+					ON_ERROR_STOP, _ := strconv.ParseBool(env.All()["ON_ERROR_STOP"])
```

Comment:
Do we have to parse it again, or is it guaranteed to have only specific values here, after the changes in `Set()`?

Reply:
The idea behind using strconv.ParseBool is to allow any values strconv.ParseBool allows, but that also means having to parse it to check it. Using QUIET as an example, it only allows the values in types.go's ParseBool and is checked via string comparison for use: https://github.com/xo/usql/blob/master/handler/handler.go#L948. This could be switched to also using type.go ParseBool and be a string check against "on" instead.